chapters/inheritance.tex

```diff
@@ -568,6 +568,7 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 
 \subsection{Removing Modifiers -- break}\label{removing-modifiers-break}
 Modifications may contain the special keyword \lstinline!break! instead of an expression.
+\index{break@\robustinline{break}}
```

Comment:
We tend to have the `\index` next to the term in the source, and make use of dedicated macro `\indexinline`:
```suggestion
Modifications may contain the special keyword \lstinline!break!\indexinline{break} instead of an expression.
```

Reply:
With more than one entry for `break`, it would also be good to explain how they differ in the index, but then the dedicated macro must be given up again:
```suggestion
Modifications may contain the special keyword \lstinline!break!\index{break@\robustinline{break}!removing modifier} instead of an expression.
```